src/app/clusters/energy-preference-server/energy-preference-server.cpp

```diff
@@ -0,0 +1,217 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "energy-preference-server.h"
+
+#include <app/util/attribute-storage.h>
```

Comment:
Again, why is this here?  The compiler error you pasted shows that you are not including _AttributeAccessInterface.h_.  Including some other header that happens to include that one today (but might not tomorrow) is not the right way to resolve that.

Reply:
see comment